src/checkers/inference/solver/frontend/Lattice.java

```diff
@@ -70,20 +57,38 @@
      */
     public final int numTypes;
 
+    /**
+     * Runtime Annotation Mirrors extracted from slots.
+     * This field is useful for type systems that has a dynamic number
+     * of type qualifiers.
+     */
+    public final Collection<AnnotationMirror> runtimeAMs;
```

Comment:
renaming: `runtime` -> `allAnnotations`, in javadoc explain the detail meaning of `all` is for recording concrete qualifiers information from the program we are running on.